lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -3573,8 +3577,29 @@ DotOpConversion::convertMMA884(triton::DotOp op, DotOpAdaptor adaptor,
         (m * 2 + 1) + (n * 4 + 2) * numM, // row3
         (m * 2 + 1) + (n * 4 + 3) * numM,
     }};
+    return idx;
+  };
+
+  { // convert the acc's value from accumuator-external layout to
```

Comment:
strictly speaking, we are not changing the layout (it's always mma_v1_layout), but only change the order of different elems in a thread. I feel the 'layout' in the comments a little bit misleading.

Reply:
I don't think so, it is the data layout of the acc. layout is a general term I think.